### PR TITLE
Feature/add artifactory helm example

### LINKED ISSUE #63: Please provide an Helm example
I could not find an example for Helm.
I'm working on it currently but if someone else has one - feel free. :-)

### PR DESCRIPTION
**Pull request by Moep90**
fixes #63
* adds a helmchart example for artifactory

### CODE REVIEW

File: inventory/targets/examples/artifactory.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+classes:
+  - common
+
+parameters:
+  target_name: artifactory
+
+  kapitan:
+    dependencies:
+    - type: https
+      source: https://charts.jfrog.io/artifactory/helm/artifactory-11.7.6.tgz
```
Comment (ademariag) [inline inventory/targets/examples/artifactory.yml:14]: is there no chart we can download with the "helm" type?

File: inventory/targets/examples/artifactory.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      unpack: True
+      output_path: components/charts
+    compile:
+    - input_type: helm
+      input_paths:
+        - components/charts/artifactory
+      output_path: .
+      helm_values:
```
Comment (ademariag) [inline inventory/targets/examples/artifactory.yml:22]: I would move this to :

```
parameters:
  artifactory:
    helm:
      values:
        name: artifactory
        <yadayada>
```

so it becomes easier to override things from other places

File: inventory/targets/examples/artifactory.yml (hunk 1 above)
Comment (Moep90) [reply inventory/targets/examples/artifactory.yml:14]: sure there is.
I just used helm as described here: https://github.com/kapicorp/kapitan/blob/master/docs/compile.md#helm

File: inventory/targets/examples/artifactory.yml (hunk 2 above)
Comment (Moep90) [reply inventory/targets/examples/artifactory.yml:22]: Sure let me update this but I used the reference above which says `helm_values:`

**Comment by Moep90** [comment]
Can we merge this or do you @ademariag missing something else?

**Pull request closed without merging**